perllib/Open311/PopulateServiceList.pm

```diff
@@ -266,7 +266,7 @@ sub _add_meta_to_contact {
     }
 
     if ( @meta ) {
-        $contact->extra( \@meta );
+        $contact->extra( { _fields => \@meta } );
```

Comment:
oh yes, `$contact->set_extra_fields(@meta)` doesn't even need the `if` (as an empty meta just sets `[]`)